pkg/kanctl/actionset.go

```diff
@@ -631,6 +664,21 @@ func verifyParams(ctx context.Context, p *PerformParams, cli kubernetes.Interfac
 		}
 	}()
 
+	// RepositoryServer
+	go func() {
+		defer wg.Done()
+		if p.RepositoryServer != nil {
+			rs, err := crCli.CrV1alpha1().RepositoryServers(p.RepositoryServer.Namespace).Get(ctx, p.RepositoryServer.Name, metav1.GetOptions{})
+			if err != nil {
```

Comment:
Should we check if the error actually says that the resource is not found. So we would have three categories
- Resource is not found
- There was an error getting the resource
- Resource is found and not ready

You don't have to do it, let's  wait for others opinions.

Reply:
- So the checks for first scenerio, ```Resource is not found``` and the third scenerio ```Resource is found and not ready``` has been handled while creating actionset, we make sure Resources are Present and Resources are in ready state
- And the second scenerio, ```There was an error getting the resource``` is handled in the code above

Reply:
Implemented !